[2021-04-20 07:00:04,634] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: global_dag.Begin_execution 2018-11-01T00:00:00+00:00 [queued]>
[2021-04-20 07:00:04,665] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: global_dag.Begin_execution 2018-11-01T00:00:00+00:00 [queued]>
[2021-04-20 07:00:04,667] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2021-04-20 07:00:04,669] {taskinstance.py:867} INFO - Starting attempt 1 of 4
[2021-04-20 07:00:04,670] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2021-04-20 07:00:04,736] {taskinstance.py:887} INFO - Executing <Task(DummyOperator): Begin_execution> on 2018-11-01T00:00:00+00:00
[2021-04-20 07:00:04,740] {standard_task_runner.py:53} INFO - Started process 3142 to run task
[2021-04-20 07:00:04,852] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: global_dag.Begin_execution 2018-11-01T00:00:00+00:00 [running]> 68b6afff5d68
[2021-04-20 07:00:04,891] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=global_dag, task_id=Begin_execution, execution_date=20181101T000000, start_date=20210420T070004, end_date=20210420T070004
[2021-04-20 07:00:14,595] {logging_mixin.py:112} INFO - [2021-04-20 07:00:14,594] {local_task_job.py:103} INFO - Task exited with return code 0
